types/nginstack__datasource/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for @nginstack/datasource 50.0
+// Project: https://dev.azure.com/nginstack/nginstack
+// Definitions by: Renato Ribeiro <https://github.com/Henato>
+//                 Paulo Moreno <https://github.com/paulocmoreno>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.3
+/// <reference path="lib/ClassDerivation.d.ts" />
```

Comment:
ClassDerivation uses exports, so the correct syntax is `export * from "lib/ClassDerivation"`, unless I'm missing something.

Reply:
fixed